runtime/doc/options.txt

```diff
@@ -4379,7 +4380,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	NOTE: This option is set when 'compatible' is set.
 
 					*'key'*
-'key'			Removed. {Nvim}
+'key'			Removed; see |vim-differences|.
```

Comment:
it's customary in vim help to just plop a tag right after a word, without using extra verbiage such as "see". (There are some exceptions). So how about this:

    'key'			Removed. |vim-differences| {Nvim}